core/src/main/java/tc/oc/pgm/commands/UserCommands.java

```diff
@@ -0,0 +1,112 @@
+package tc.oc.pgm.commands;
+
+import app.ashcon.intake.Command;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+import net.md_5.bungee.api.ChatColor;
+import org.bukkit.command.CommandSender;
+import tc.oc.pgm.api.Permissions;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.api.player.MatchPlayer;
+import tc.oc.pgm.community.features.VanishManager;
+import tc.oc.pgm.teams.TeamMatchModule;
+import tc.oc.pgm.util.component.Component;
+import tc.oc.pgm.util.component.Components;
+import tc.oc.pgm.util.component.types.PersonalizedText;
+import tc.oc.pgm.util.component.types.PersonalizedTranslatable;
+import tc.oc.pgm.util.named.NameStyle;
+
+public class UserCommands {
```

Comment:
`UserCommands` feels so generic many other commands could fit in here, what do you think about `ListCommands`?